README.md

```diff
@@ -1,24 +1,52 @@
+
 # IPsec-Recipe
+
 The IPsec recipe is an application that enables strongSwan to use [Infrastructure Application Interface](https://ipdk.io/documentation/Interfaces/InfraApp/), specifically the P4Runtime and OpenConfig gRPCs provided by the [Networking-Recipe](https://github.com/ipdk-io/networking-recipe).
 
 The strongSwan plugin available in this repository implements a policy-based IPsec enablement using security policy database (SPD) and security association database (SAD). The P4Runtime client introduces a programmable IPsec flow using P4 language for the SPD and OpenConfig for SAD configurability.
 
 * IPsec recipe is validated only on Intel&reg; IPU E2100 target
 * Refer to https://ipdk.io/documentation/Recipes/InlineIPsec/
-* YANG model for IPsec SAD: https://github.com/ipdk-io/openconfig-public/blob/master/release/models/ipsec/openconfig-ipsec-offload.yang
+* YANG model for IPsec offload: https://github.com/ipdk-io/openconfig-public/blob/master/release/models/ipsec/openconfig-ipsec-offload.yang
 * Reference P4 program to enable IPsec on DPDK target: https://github.com/ipdk-io/networking-recipe/tree/main/p4src/Inline_IPsec
 
-strongSwan Integration with Inline Crypto Engine
-========================================================
+## strongSwan Integration with Inline Crypto Engine
+
 * The user should start and stop the strongSwan services as described in strongSwan manual.
-* The strongSwan configuration files must be configured by user as described in 
+* The strongSwan configuration files must be configured by user as described in
 			https://wiki.strongswan.org/projects/strongswan/wiki/UserDocumentation
 * Confirm that the networking-recipe (the `infrap4d` process, which acts as the gRPC server) is running before starting strongSwan services.
-* strongSwan hosts (Host-A<---->Host-B) should be connected to negotiate the keys.
-* The user should follow best practices followed by opensource community for other control plane related configurations.
- 
-Getting Started
-===================
+* strongSwan hosts (Host-A<---->Host-B) must be connected to negotiate the keys.
+* The user must follow best practices followed by opensource community for other control plane related configurations.
```

Comment:
```suggestion
* You should follow the best practices of the open-source community for other control plane related configurations.
```
- You're addressing the user. Referring to the reader in the third person is distant and formal. "You" is much friendlier.
- "Best practices" are preferred, not mandatory, so one "_should_ follow," rather than "_must_ follow."
- This avoids saying "**follow** the best practices **followed**," which is redundant.
- "Open source" is two words. They should generally be hyphenated when used as a modifier. (This helps the reader parse it as _"open source" community_, as opposed to _open "source community."_) The day will doubtless come when the two are routinely written as one word, but I don't think we're quite there yet.